odkx-src/notify-intro.rst

```diff
@@ -0,0 +1,14 @@
+ODK-X Notify
```

Comment:
We have moved away from having the separate intro pages in the docs -- suggest integrating this at the top of the install. See, for example, https://docs.odk-x.org/services-using/
Otherwise people will do exactly what I did when commenting and miss the page and not understand :)

Reply:
Combined the intro and installation pages.